File: src/main/java/calebzhou/rdi/microservice/model/json/loca/AdInfo.java

package calebzhou.rdi.microservice.model.json.loca;

import javax.annotation.Generated;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

@Generated("jsonschema2pojo")
public class AdInfo {

    @SerializedName("adcode")
    @Expose
    private String adcode;

    public String getAdcode() {
        return adcode;
    }

    public void setAdcode(String adcode) {
        this.adcode = adcode;
    }

}

File: src/main/java/calebzhou/rdi/microservice/model/json/tencentlbs/ip/AdInfo.kt
package calebzhou.rdi.microservice.model.json.tencentlbs.ip

import com.google.gson.annotations.SerializedName


data class AdInfo (

  @SerializedName("nation"   ) var nation   : String? = null,
  @SerializedName("province" ) var province : String? = null,
  @SerializedName("city"     ) var city     : String? = null,
  @SerializedName("district" ) var district : String? = null,
  @SerializedName("adcode"   ) var adcode   : Int?    = null

)